### PR TITLE
2124 admin add actual total distance audited

### LINKED ISSUE #2124: Include total distance audited on admin user page
On the user dashboard it shows the total distance you've audited. I'd really like to have that on the /admin/user pages as well.

### PR DESCRIPTION
**Pull request by nomichka**
Resolves #2124: adds Total Distance Audited to the table on the admin/user/<username> dashboard. This PR makes the requested changes from the previous PR on this branch - fixes the width of the table, and adds unit conversions for mi or km depending on the admin's set language on their browser.

### CODE REVIEW

**Comment by misaugstad** [comment]
Thanks @nomichka ! Can you resolve the merge conflicts with the develop branch and make sure it still works?

When you say "resolves X" can you include the "#" before the number? That creates a link in Github directly to the issue.

**Comment by misaugstad** [comment]
When testing this, the spot in the table for the distance was blank, and I got this error in the browser console:
```
Uncaught TypeError: Cannot set property 'innerHTML' of null
```

**Comment by nomichka** [comment]
:open_mouth: I must've accidentally deleted the id tag. I added it and committed just now, and it should work.

**Comment by misaugstad** [comment]
Okay, that worked, thank you!

File: app/views/admin/user.scala.html [hunk 1]
```diff
@@ -4,10 +4,12 @@
 @import models.audit.AuditTaskTable
 @import models.label.LabelTable
 @import models.mission.MissionTable
+@import models.region.RegionTable
```
Comment (misaugstad) [inline app/views/admin/user.scala.html:7]: You don't use either of these added imports anywhere do you? Just a copy/paste error maybe?

Comment (nomichka) [reply app/views/admin/user.scala.html]: Good catch! The two imports were just remnants of what I tried earlier. I deleted them, and everything still works.

**Pull request merged**